src/components/Home/index.js

```diff
@@ -0,0 +1,106 @@
+/* eslint-disable camelcase */
+import React, { useEffect, useState } from 'react';
```

Comment:
- [ ] Kindly avoid disabling the linter since it enforces rules for quality code. Instead, you should fix the raised issues.

Reply:
Resolved

Reply:
Eslint restored, issues resolved